2024-01-19 15:57:11,064 INFO    Thread-101 (_run_job):76495 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-01-19 15:57:11,064 INFO    Thread-101 (_run_job):76495 [wandb_setup.py:_flush():76] Configure stats pid to 76495
2024-01-19 15:57:11,064 INFO    Thread-101 (_run_job):76495 [wandb_setup.py:_flush():76] Loading settings from /Users/adamimos/.config/wandb/settings
2024-01-19 15:57:11,064 INFO    Thread-101 (_run_job):76495 [wandb_setup.py:_flush():76] Loading settings from /Users/adamimos/Documents/GitHub/epsilon-transformers/wandb/settings
2024-01-19 15:57:11,064 INFO    Thread-101 (_run_job):76495 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'epsilon-transformer', 'entity': 'adamimos', 'root_dir': '/Users/adamimos/Documents/GitHub/epsilon-transformers', 'run_id': '1qusijnn', 'sweep_param_path': '/Users/adamimos/Documents/GitHub/epsilon-transformers/wandb/sweep-eaedbzxy/config-1qusijnn.yaml', 'sweep_id': 'eaedbzxy'}
2024-01-19 15:57:11,064 INFO    Thread-101 (_run_job):76495 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-19 15:57:11,064 INFO    Thread-101 (_run_job):76495 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-19 15:57:11,064 INFO    Thread-101 (_run_job):76495 [wandb_init.py:_log_setup():524] Logging user logs to /Users/adamimos/Documents/GitHub/epsilon-transformers/wandb/run-20240119_155711-1qusijnn/logs/debug.log
2024-01-19 15:57:11,064 INFO    Thread-101 (_run_job):76495 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/adamimos/Documents/GitHub/epsilon-transformers/wandb/run-20240119_155711-1qusijnn/logs/debug-internal.log
2024-01-19 15:57:11,065 INFO    Thread-101 (_run_job):76495 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x2b4b18650>
2024-01-19 15:57:11,065 INFO    Thread-101 (_run_job):76495 [wandb_init.py:init():564] calling init triggers
2024-01-19 15:57:11,065 INFO    Thread-101 (_run_job):76495 [wandb_init.py:init():571] wandb.init called with sweep_config: {'learning_rate': 0.0004641588833612778, 'weight_decay': 0}
config: {}
2024-01-19 15:57:11,065 INFO    Thread-101 (_run_job):76495 [wandb_init.py:init():614] starting backend
2024-01-19 15:57:11,065 INFO    Thread-101 (_run_job):76495 [wandb_init.py:init():618] setting up manager
2024-01-19 15:57:11,071 INFO    Thread-101 (_run_job):76495 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-01-19 15:57:11,071 INFO    Thread-101 (_run_job):76495 [wandb_init.py:init():624] backend started and connected
2024-01-19 15:57:11,083 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_config_callback():1342] config_cb None None {'learning_rate': 0.0004641588833612778, 'weight_decay': 0}
2024-01-19 15:57:11,115 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-01-19 15:57:11,115 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-01-19 15:57:11,115 INFO    Thread-101 (_run_job):76495 [wandb_init.py:init():716] updated telemetry
2024-01-19 15:57:11,138 INFO    Thread-101 (_run_job):76495 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-01-19 15:57:11,365 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_on_init():2254] communicating current version
2024-01-19 15:57:11,440 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.16.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-01-19 15:57:11,440 INFO    Thread-101 (_run_job):76495 [wandb_init.py:init():800] starting run threads in backend
2024-01-19 15:57:12,806 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_console_start():2233] atexit reg
2024-01-19 15:57:12,806 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-01-19 15:57:12,806 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-01-19 15:57:12,806 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_redirect():2178] Redirects installed.
2024-01-19 15:57:12,807 INFO    Thread-101 (_run_job):76495 [wandb_init.py:init():841] run started, returning control to user process
2024-01-19 15:57:13,281 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_config_callback():1342] config_cb None None {'model_config': {'n_layers': 2, 'd_model': 16, 'n_ctx': 10, 'd_head': 4, 'model_name': 'custom', 'n_heads': 4, 'd_mlp': 64, 'act_fn': 'relu', 'd_vocab': 2, 'eps': 1e-05, 'use_attn_result': False, 'use_attn_scale': True, 'use_split_qkv_input': False, 'use_hook_mlp_in': False, 'use_attn_in': False, 'use_local_attn': False, 'original_architecture': None, 'from_checkpoint': False, 'checkpoint_index': None, 'checkpoint_label_type': None, 'checkpoint_value': None, 'tokenizer_name': None, 'window_size': None, 'attn_types': None, 'init_mode': 'gpt2', 'normalization_type': None, 'device': 'cpu', 'n_devices': 1, 'attention_dir': 'causal', 'attn_only': False, 'seed': 42, 'initializer_range': 0.2, 'init_weights': True, 'scale_attn_by_inverse_layer_idx': False, 'positional_embedding_type': 'standard', 'final_rms': False, 'd_vocab_out': 2, 'parallel_attn_mlp': False, 'rotary_dim': None, 'n_params': 6144, 'use_hook_tokens': False, 'gated_mlp': False, 'default_prepend_bos': True, 'dtype': 'torch.float32', 'tokenizer_prepends_bos': None, 'post_embedding_ln': False}, 'train_config': {'batch_size': 64, 'sequence_length': 500, 'num_sequences': 100, 'num_epochs': 200, 'learning_rate': 0.00015, 'weight_decay': 0}}
2024-01-19 16:12:11,934 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_finish():1962] finishing run adamimos/epsilon-transformer/1qusijnn
2024-01-19 16:12:11,934 INFO    Thread-101 (_run_job):76495 [jupyter.py:save_history():445] not saving jupyter history
2024-01-19 16:12:11,934 INFO    Thread-101 (_run_job):76495 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-19 16:12:11,934 INFO    Thread-101 (_run_job):76495 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-01-19 16:12:11,934 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-19 16:12:11,935 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_restore():2185] restore
2024-01-19 16:12:11,935 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_restore():2191] restore done
2024-01-19 16:12:19,871 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_footer_history_summary_info():3837] rendering history
2024-01-19 16:12:19,873 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_footer_history_summary_info():3869] rendering summary
2024-01-19 16:12:19,885 INFO    Thread-101 (_run_job):76495 [wandb_run.py:_footer_sync_info():3796] logging synced files
